DownloadManager.sln

```diff
@@ -414,6 +415,9 @@ Global
 		{D32EDF26-94F4-4F80-B26C-CA250A3F6D7D} = {11FE7FBB-8560-4D89-90F5-F218CF18A879}
 		{FEE6A3A2-9CDD-473C-8587-6C13241BD03A} = {11FE7FBB-8560-4D89-90F5-F218CF18A879}
 	EndGlobalSection
+	GlobalSection(ExtensibilityGlobals) = postSolution

```

Comment:
What is this change for? What does it help - what does it do?

Reply:
I haven't change anything regarding this, VS 2017 v15.9.9.9 is the one to blame.
It could be added when I set the Android project to deploy.